students/17th/kangminji/styles/instagram.css

```
@@ -0,0 +1,351 @@
+*{
+    box-sizing: border-box;
+    margin:0;
+    padding:0;
+    background-color: rgb(249 249 249);
+
+}
+.nav{
+position: sticky;
+top: 0;
+    font-family: 'Lobster', cursive;
+    font-size: 30px;
+    width: 100%;
+    height: 54px;
+    display : flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    border-top: solid 1px black;
+    border-bottom: 1px solid rgb(160 163 165);
+    background-color: white;
+    z-index: 100;
+   
+ }
+
+ .markleft{
+    display: flex;
+    align-self: flex-end;
+    margin-left: 30px;
+    
+    }
+
+.navcenter{
+    display : flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    width: 85%;
+    background-color: white;    
+}
+.headmark{
+    width: 30px;
+    height: 30px;
+    }
+.l{
+    margin-left:50px ;
+    margin-top: 10px;
+}
+.instahead {
+  
+        border-left: black 2px solid;
+        margin-left: 15px;
+       margin-bottom: 5px; 
+        margin-top: 4px;
+        padding-left: 10px;
+    }
+.markright{
+display: flex;
+margin-right: 100px;
+background-color: white;
+
+    
+}
+.center{
+width : 180px;
+background-color: white;
+    
+
+}
+.search {
+    width : 150px;
+    text-align: center;
+}
+
+/* .markleft{
+display: flex;
+align-self: flex-end;
+margin-left: 30px;
+
+} */
+
+.right{
+    margin-right: 30px;
+    margin-top: 10px;
+}
+
+
+.container{
+    position: relative;
+
+width: 100%;
+height: 900px;
+}
+
+.feeds{
+position: relative;
+width: 600px;
+
+height: auto;
+border: 1px solid rgb(160 163 165);
+
+border-radius: 3px;
+
+}
+
+.feed{
+position: relative;
+width: 90%;
+height: auto;
+border: 1px solid rgb(160 163 165);
+border-radius: 3px;
+margin-left: 30px;
+margin-bottom: 90px;
+top : 22px;
+padding-bottom: 10px;
+}
+
+#comment{
+    position: absolute;
+width : 90%;
+height: 45px;
+left: 30px;
+bottom : 10px;
+}
+
+.content-right{
+
+width : 300px; 
+height : 700px;
+margin-left: 40px;
+margin-right: 10px;
+
+
+}
+
+.rightone{
+    margin-left: -5px;
+    margin-bottom: 15px;
+}
+
+.profile{
+   
+    width: 50px;
+    height: 50px;
+    border-radius: 50px;
+    margin-left: 10px;
+    margin-top: 10px;
+    
+}
+
+.head{
+
+ position: absolute;
+    left: 70px;
+    top : 13px;
+    font-size: 15px;
+    margin: auto;
+}
+
+.contentpicture {
+
+  
+    width : 100%;
+    height: 650px;
+    /* top: 30px; */
+   margin-top: 5px; 
+}
+
+.me{
+   
+    width : 50px;
+    height: 50px;
+    border-radius: 50px;
+  margin-top: 10px;
+  margin-left: 15px;
+}
+
+.profileme{
+position: absolute;
+margin-top: 20px;
+margin-left: 10px;
```

아래와 같이 한 줄로 작성하여 코드를 줄일 수 있습니다.
```suggestion
margin: 20px 0 0 10px;
```

---

넵 수정하겠습니다!